### PR TITLE
Adding summary pdfs to the output data products.

### PR DESCRIPTION
**Pull request by cmccully**

### CODE REVIEW

File: banzai_nres/frames.py [hunk 1]
```diff
@@ -62,26 +63,69 @@ def table(self):
...
+            output_product_2d = DataProduct.from_fits(fits_2d, filename_2d, self.get_output_directory(runtime_context))
+
+            summary_pdf_meta = fits_utils.sanitize_header(self.meta.copy())
+            for keyword in ['NAXIS1', 'NAXIS2', 'EXTNAME', 'EXTVER']:
+                if keyword in summary_pdf_meta:
+                    summary_pdf_meta.pop(keyword)
+
+            summary_buffer = BytesIO()
+            pp = PdfPages(summary_buffer, keep_empty=False, metadata=dict(summary_pdf_meta))
+            # make sure we pop all the fits garbage before we write the meta data.
```
Comment (mgdaily) [inline banzai_nres/frames.py:117]: this comment seems out of place? is that supposed to go above the keyword popping logic you have above?

**Review by mgdaily — APPROVED** [review]
New PDF stuff looks good, had a question about a comment you added.

**Review by gmbrandt — COMMENTED** [review]
Looks good, just need to merge in feature/split_products again.

File: banzai_nres/frames.py (hunk 1 above)
Comment (cmccully) [reply banzai_nres/frames.py]: Fixed.

**Pull request merged**